snapshot/snapshot.go

```diff
@@ -736,14 +744,18 @@ func (o *snapshotter) checkAvailability(ctx context.Context, key string) bool {
 }
 
 func (o *snapshotter) restoreRemoteSnapshot(ctx context.Context) error {
-	mounts, err := mountinfo.GetMounts(nil)
-	if err != nil {
-		return err
-	}
-	for _, m := range mounts {
-		if strings.HasPrefix(m.Mountpoint, filepath.Join(o.root, "snapshots")) {
-			if err := syscall.Unmount(m.Mountpoint, syscall.MNT_FORCE); err != nil {
-				return fmt.Errorf("failed to unmount %s: %w", m.Mountpoint, err)
+	// In detach mode, rs is taken over by fusemanager,
+	// and there may be running containers, so we skip clean
```

Comment:
No need to mention about fusemanager as snapshot/snapshot.go is agnostic about fusemanager.

Reply:
done